addOns/reports/src/main/java/org/zaproxy/addon/reports/ReportDialog.java

```diff
@@ -278,6 +279,22 @@ public void changedUpdate(DocumentEvent e) {
         this.pack();
     }
 
+    private static JCheckBox getDisplayCheckbox(boolean selected) {
+        JCheckBox displayCheckbox = new JCheckBox();
+        displayCheckbox.setSelected(selected);
+        if (isDocker()) {
+            displayCheckbox.setSelected(false);
+            displayCheckbox.setEnabled(false);
+        }
```

Comment:
I know this could have been done with `!isDocker && selected`. But I thought it read more obviously this way. I know it means the control might be set/reset in some situations but that's very low overhead.